packages/ui/src/documents/FundingAgreements.tsx

```diff
@@ -54,48 +63,39 @@ export const FundingAgreements: FunctionComponent<Props> = (props) => {
     user,
     contacts,
     viewMode,
-  } =
-    {
-      onAsyncStart: (message: string) => {
-      },
-      onAsyncComplete: (data) => {
-      },
-      onAsyncError: (error, title?: string) => {
-      },
+  } = {
+    onAsyncStart: (message: string) => {},
+    onAsyncComplete: data => {},
+    onAsyncError: (error, title?: string) => {},
```

Comment:
Are these just no-ops?

Reply:
Yes, but these changes are just here because of formatting that my editor did on save. Please ignor.